pkg/pillar/types/errortime.go

```diff
@@ -12,19 +12,93 @@ import (
 
 // Common error with timestamp
 
+// ErrorSeverity tells the severity type, must be in sync with info api
```

Comment:
Would be good to state the name of the enum in the info API.

Reply:
I added to the comment `must be in sync with info.Entity enum of api`